Typing Part 3: events.py

herostrat opened the pull request.

NickM-27 commented on frigate/events.py:
```diff
@@ -147,14 +158,15 @@ def run(self):
 
 class EventCleanup(threading.Thread):
-    def __init__(self, config: FrigateConfig, stop_event):
+    def __init__(self, config: FrigateConfig, stop_event: Event):
```
stop_event is a multiprocessing event, I think this would make this typed as the Frigate Event db model

herostrat replied:
I think you are right.
The ambiguity is kind of dangerous...
See:
https://github.com/blakeblackshear/frigate/blob/1d45b0b3519dc7ce2e3c3a11747c61aa52605efc/frigate/app.py#L5
https://github.com/blakeblackshear/frigate/blob/1d45b0b3519dc7ce2e3c3a11747c61aa52605efc/frigate/app.py#L28
https://github.com/blakeblackshear/frigate/blob/1d45b0b3519dc7ce2e3c3a11747c61aa52605efc/frigate/app.py#L48

herostrat replied:
What do you think, should we rename the models to FrigateEvent and FriagteRecordings?

NickM-27 replied:
Yeah, that is ambiguous. I was thinking since the Frigate Event and Recordings are much more common, it may be more straightforward to alias the other `Event` types

NickM-27 replied:
It seems like `from multiprocessing.synchronize import Event` may just be used for typing, could be `as ProcessEvent` or something 🤔

Commit:
Typing: events.py

Commit:
Remove unused variable

Commit:
Fix return Any from return statement

Not all elements from the event dict are sure to be something that can be evaluated

See e.g.: https://github.com/python/mypy/issues/5697

NickM-27 commented:
bump

herostrat committed:
Sort out Event disambiguity

There was a name collision of multiprocessing Event type and frigate events

herostrat commented:
Sorry for taking so long to resolve the error.
Imho this is now good to go.
